opensrp-app/src/test/java/org/smartregister/repository/ClientRelationshipRepositoryTest.java

```diff
@@ -54,5 +60,35 @@ private Cursor getClients() throws JSONException {
         return matrixCursor;
     }
 
+    @Test
+    public void testRawQueryShouldReturnClientDetails() throws JSONException {
+        ClientRelationshipRepository repository = spy(ClientRelationshipRepository.class);
+        Cursor cursor = spy(getClients());
+        when(database.rawQuery(anyString(), any()))
+                .thenReturn(cursor);
+        String jsonFieldColumn = "json";
+        String query = String.format("SELECT %s FROM client_relationship;", jsonFieldColumn);
+
+        List<HashMap<String, String>> result = repository.rawQuery(database, query);
```

Comment:
When is this executed? Can't determine that from the ClientRelationshipRepository class

Reply:
It's a method in BaseRepository class, the base class for ClientRelationshipRepository class.
the test is not really covered with `findClientByRelationshipShouldReturnClients` test, since the `findClientByRelationshipShouldReturnClients` calls directly to database.rawquery and not the base class method.
Reason `testRawQueryShouldReturnClientDetails` was part of ClientRelationshipRepositoryTest, instead of BaseRepository, was just a matter of convenience